antlr-kotlin-runtime/src/androidNativeMain/kotlin/com/strumenta/antlrkotlin/runtime/Console.kt

```diff
@@ -0,0 +1,10 @@
+// Copyright 2017-present Strumenta and contributors, licensed under Apache 2.0.
+// Copyright 2024-present Strumenta and contributors, licensed under BSD 3-Clause.
```

Comment:
For new files, should we keep only the first line?

Reply:
Don't you still release it with double license? Or do you want to drop Apache completely?

Reply:
Ah sorry, the opposite. Do you only want Apache licensing?

Reply:
Sorry, you are right. It is still available under both licenses